graphene/sys/src/lib.rs

```diff
@@ -62,9 +62,6 @@ pub const GRAPHENE_RAY_INTERSECTION_KIND_ENTER: graphene_ray_intersection_kind_t
 pub const GRAPHENE_RAY_INTERSECTION_KIND_LEAVE: graphene_ray_intersection_kind_t = 2;
 
 // Constants
-pub const GRAPHENE_HAS_GCC: c_int = 1;
-pub const GRAPHENE_HAS_SCALAR: c_int = 1;
-pub const GRAPHENE_HAS_SSE: c_int = 1;
 pub const GRAPHENE_PI: c_double = 3.141593;
 pub const GRAPHENE_PI_2: c_double = 1.570796;
```

Comment:
I would probably just ignore those & make them an alias of the std types

Reply:
Yeah, like I wrote in the gir PR :)